.github/workflows/lgtm.yml

```
@@ -0,0 +1,52 @@
+    workflows: ["Checking workflow files"]   
+    types: [completed]
+
+jobs:
+  add-lgtm:
+    if: ${{ github.event.workflow_run.event == 'pull_request' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get PR number
+        id: pr
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)